contracts/lib.rs

```diff
@@ -573,7 +545,7 @@ pub mod prosopo {
             self.check_provider_fee(fee)?;
 
             // provider cannot be an operator
```

Comment:
Aren't operators removed?

Reply:
Ah, it's just the comment, function name says admin

Reply:
I will adjust the comment, missed it :)

Reply:
amended